packages/x-tailwindcss/src/x-tailwind-plugin/components/suggestion-group/outlined.ts

```diff
@@ -0,0 +1,23 @@
+import { TailwindHelpers } from '../../../types';
+
+/**
+ * Returns the `outlined` variant for component `suggestion group`.
+ *
+ * @param helpers - The {@link TailwindHelpers} to generate CSS.
+ * @returns The {@link CssStyleOptions} for the variant.
+ */
+// eslint-disable-next-line  @typescript-eslint/explicit-function-return-type
+export function suggestionGroupOutlined(helpers: TailwindHelpers) {
+  const { theme } = helpers;
+  return {
+    outlined: {
+      backgroundColor: theme('colors.neutral.0'),
+      borderWidth: theme('borderWidth.1'),
+      borderColor: 'var(--suggestion-group-color-50)',
```

Comment:
issue: This variable does not have to be defined unless you use one of the color classes. Should we add a fallback value from the theme like the figma file says with `neutral-90`?

![image](https://user-images.githubusercontent.com/68222542/209780783-845af367-e1d0-494c-9a7b-89b9dff01869.png)

Reply:
yes, you are right! Besides, I noticed that the border color should point to `--suggestion-group-color-75` and not `--suggestion-group-color-50`